@stellar/anchor-tests/src/helpers/console.ts

```diff
@@ -64,6 +64,19 @@ async function printColoredTextTestRun(testRun: TestRun, verbose: boolean) {
       console.log(`Received: ${testRun.result.actual}\n`);
     }
     console.groupEnd(); // description group
+    if (testRun.result.failure.links) {
+      const resources = Array.from(
```

Comment:
`Object.entries` already returns an array, so you don't actually need to use `Array.from` here

Reply:
fixed 👍